src/components/data-management/ProjectDetails.jsx

```diff
@@ -341,6 +326,46 @@ const ProjectDetails = ({ width, height }) => {
     dispatch(deleteMetadataTrack(name, activeProjectUuid));
   };
 
+  const updateSpeciesOrMetadata = (value, metadataKey) => {
+    const updateObject = metadataKey === 'species' ? { species: value } : { metadata: { [metadataKey]: value } };
+    return updateObject;
+  };
+
+  const replaceEmptyCells = (value, metadataKey) => {
+    const update = updateSpeciesOrMetadata(value, metadataKey);
```

Comment:
the name `update` is confusing -- I would change it to a name that shows that the variable holds the name of the thing that will be updated. For example, I think `typeToUpdate` or `updateType` or something similar will be better

Reply:
or `sampleType`? Because of line 345

Reply:
I'll change it to `updateObject`